lib/client-assets.php

```diff
@@ -866,11 +866,26 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	 */
 	$allowed_block_types = apply_filters( 'allowed_block_types', true );
 
+	// Remove the default media buttion.
+	remove_action( 'media_buttons', 'media_buttons' );
+	ob_start();
+	/**
+	 * Fires after the default media button(s) are displayed.
+	 *
+	 * @since 2.5.0
+	 *
+	 * @param string $editor_id Unique editor identifier, e.g. 'content'.
+	 */
+	do_action( 'media_buttons', 'content' );
```

Comment:
Is the `global $post` set by this point?

Reply:
yes, we're already using it a bit above